mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1649,6 +1649,109 @@ export const getActions = (
             searchTerms: ['docs', 'documentation', 'help', 'support'],
             tooltip: "Documentation, tutorials, and how-tos on all functionality in Mito."
         },
+        [ActionEnum.Select_Columns]: {
+            type: 'build-time',
+            staticType: ActionEnum.Select_Columns,
+            longTitle: 'Select column',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                // We close the editing taskpane if its open
+                closeOpenEditingPopups();
+
+                // We open the column selection taskpane
+                let newStartingColumnIndex = gridState.selections[0].startingColumnIndex;
+                let newEndingColumnIndex = gridState.selections[0].endingColumnIndex;
+                if (newStartingColumnIndex === -1) {
+                    newStartingColumnIndex = 0;
+                }
+                if (newEndingColumnIndex === -1) {
+                    newEndingColumnIndex = sheetData.numColumns - 1;
+                }
+                setGridState(prevGridState => {
+                    return {
+                        ...prevGridState,
+                        selections: [{
+                            startingColumnIndex: newStartingColumnIndex ?? 0,
+                            endingColumnIndex: newEndingColumnIndex ?? 0,
```

Comment:
It feels like ?? is actually a strange operator here, given that we're dealing with numbers. 

Let's make the undefined check explicit (e.g. `== undefined ? ...`) to avoid any unexpected behavior, and make this easier to reason about.

Reply:
Oh yeah this is actually not even necessary anymore